game/core/gui/message_box.py

```diff
@@ -10,7 +10,7 @@ class MessageBox:
 
     message: str = ""
     notes: str = ""
-    _message_box_width = 500
+    _message_box_width = 600
     _message_box_height = 60
     _message_font = 12
     _notes_font = 8
```

Comment:
Can the note text be larger? It is hard to read on my laptop

Locally, these sizes look good:

```suggestion
    _message_box_width = 700
    _message_box_height = 90
    _message_font = 14
    _notes_font = 12
```

Reply:
The current sizes look fine on my laptop but I'm totally fine with having larger sizes. I can apply these changes but 10 instead of 12, since the 14 and 12 look too similar